easybuild/easyconfigs/m/MXNet/MXNet-0.9.3-foss-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'MakeCp'
+
+name = 'MXNet'
+version = '0.9.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://mxnet.io/' 
+description = """Flexible and Efficient Library for Deep Learning"""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'cstd': 'c++11', 'opt': True, 'pic': True, 'openmp': True}
+
+#Download source from github, fix sub branches, checkout to version, 
```

Comment:
style nitpicking: single space after the `#` (we'll start enforcing that soon with automatic style tests)